poi.go

```diff
@@ -101,9 +101,9 @@ func GetBranches(conn Connection) ([]Branch, error) {
 }
 
 func getHostname(remoteName string) string {
-	r := regexp.MustCompile("(?:@|//)(.+):")
+	r := regexp.MustCompile("(@|//)(.+?)(:|/)")
```

Comment:
We are only interested in the hostname, so we don't need to capture the group.
In that case, we can leave the `found[1]` code as it is.
```suggestion
	r := regexp.MustCompile("(?:@|//)(.+?)(?::|/)")
```

Reply:
@seachicken 
I did not know the no-capturing group. Thanks :)